tests/features/document.feature

```diff
@@ -0,0 +1,51 @@
+@api
+Feature: Document media paragraph.
+  As a content editor
+  I need to be able to use the Document paragraph
+  so that I can attach documents to a content.
+  
+  @cleanup:media
+  Scenario: Document creation.
+    Given I am logged in as a user with the "create oe_demo_landing_page content, edit any oe_demo_landing_page content, access content overview, create document media, delete any document media, update any media" permission
+    
+    # Create a local "Document" media.
+    When I go to "the document creation page"
+    And I fill in "Name" with "My example local document 1"
+    And I select "Local" from "File Type"
+    And I attach the file "example_1.pdf" to "File"
+    And I press "Save"
+    Then I should see "Document My example local document 1 has been created."
+    
+    # Create a remote "Document" media.
+    When I go to "the document creation page"
+    And I fill in "Name" with "My example remote document 1"
+    And I select "Remote" from "File Type"
+    And I fill in "URL" with "http://www.africau.edu/images/default/sample.pdf"
```

Comment:
Can we use a static fixture to avoid issues with files being moved? Actually I see there's one attached to this pr.

Reply:
I've updated the remote file to the absolute path to the fixture, works fine.